exchanges/README.md

```diff
@@ -42,9 +42,6 @@ When submitting a PR, please abide by our coding guidelines:
 + Code must adhere to our [coding style](https://github.com/thrasher-corp/gocryptotrader/blob/master/doc/coding_style.md).
 + Pull requests need to be based on and opened against the `master` branch.
 
-If contributing websocket improvements, please make sure order reports
-follow [these rules](../docs/WS_ORDER_EVENTS.md).
-
```

Comment:
Can reinstate this, but I have it included on another PR.

Reply:
Will merge the state PR first, before this one